fabric-registry-sync-v0/src/main/java/net/fabricmc/fabric/impl/registry/sync/RegistrySyncManager.java

```diff
@@ -216,4 +278,30 @@ public static void unmap() throws RemapException {
 			}
 		}
 	}
+
+	public static void bootstrapRegistries() {
+		for (MutableRegistry<?> registry : Registry.REGISTRIES) {
+			if (registry instanceof HashedRegistry) {
+				((HashedRegistry) registry).storeHash();
+			}
+		}
+
+		postBootstrap = true;
+	}
+
+	// Checks the stored hash against the current hash, if it has changed mark as modded
+	public static void checkRegistryHashes() {
```

Comment:
From what I understand, the purpose of this check is to handle implementations of `MutableRegistry` which are not `SimpleRegistry`s or to handle `SimpleRegistry`s modified through accessor mixins. Otherwise, surely what is in `MixinSimpleRegistry` is sufficient. I would like to point out that all vanilla registries which are `MutableRegistry`s are also `SimpleRegistry`s.